1.9/deploying-services/creating-services/health-checks.md

```diff
@@ -1,23 +1,23 @@
 ---
-post_title: Marathon Health Checks
+post_title: Health Checks
 menu_order: 1
 ---
 
 You can define health checks for your DC/OS services. Health checks are defined on a per application basis, and are run against that application's tasks.
 
-Marathon health checks perform periodic checks on the containers distributed across a cluster to make sure they’re up and responding. If a container is down for any reason, Marathon will detect this and stop sending traffic to the container and try to restart it.
+Health checks perform periodic checks on the containers distributed across a cluster to make sure they’re up and responding. If health checks fail for any reason, Mesos will report the task as unhealthy so that status-aware load-balancers can stop sending traffic to the container. Additionally, after the configured maximum consecutive failures, Marathon will kill the task and restart it.
 
-Health checks begin immediately when a task is launched. By default, health checks are run using the Marathon scheduler. Health checks with the Marathon scheduler only support the COMMAND protocol and each app is limited to a single health check.
+Health checks begin immediately when a task is launched. They are locally executed by Mesos on the agent running the corresponding task. Health checks are performed as close to the task as possible, so they are are not affected by networking failures. Health checks delegated to the agents running the tasks, so the number of tasks that can be health checked can scale horizontally with the number of agents in the cluster.
```

Comment:
run on sentence. please revise: `Health checks are delegated to the agents running the tasks. This allows the number of tasks that are health checked to scale horizontally along with the number of agents in the cluster.`